ui/src/index.test.tsx

```
@@ -144,18 +144,34 @@ const addFriend = async (page: Page, friend: string) => {
   await page.waitForSelector('.test-select-add-friend-input > :not(.loading)');
 }
 
-// Send a message to the first friend appearing in the dropdown menu.
-const sendMessageToFirst = async (page: Page, content: string) => {
-  // Select the first option in the dropdown menu.
+// Send a message to the given user.
+// Does nothing if the user does not appear in the dropdown menu.
```

Since these are tests we should throw an exception and fail loudly if we're trying to send a message to somebody non-existent.

---

True, will do